docs/source/run/parameters.rst

```diff
@@ -354,6 +354,15 @@ When both are specified, the per-species value is used.
     arrays of size ``sort_bin_size`` (+ guard cells) that are atomic-added to the main current
     arrays.
 
+* ``<plasma name> or plasmas.reorder_interval`` (`int`) optional (default `0`)
+    Reorder particles periodically to speed-up current deposition on GPU for a high-temperature plasma.
+    An interval of 1 means reordering is done on every slice.
+    To disable reordering set this to 0.
```

Comment:
Could you provide a suggestion for a useful parameter? Was it around 5 or 10?

Reply:
On the same note: let's try to be consistent with other similar parameters, using `0` to disable reordering and `-1` to disable output is not too nice. So, do we agree that `period = 0` means disable everywhere?
Also, let's try to be consistent with naming, i.e. `period` vs. `interval`.

Reply:
The plan is to make output_period = 0 disable output.